packages/input-selection/src/selectionConstraints.ts

```diff
@@ -55,10 +55,14 @@ export const computeMinimumCoinQuantity =
   (output) =>
     usingAutoFree((scope) =>
       BigInt(
-        CML.min_ada_required(
-          coreToCml.txOut(scope, output),
-          scope.manage(CML.BigNum.from_str(coinsPerUtxoByte.toString()))
-        ).to_str()
+        scope
+          .manage(
+            CML.min_ada_required(
+              scope.manage(coreToCml.txOut(scope, output)),
```

Comment:
We've had the convention of managing any objects within the `coreTo*` translators. Can it not be done using the scope passed in?

Reply:
OK, I can see it's already managed internally here: https://github.com/input-output-hk/cardano-js-sdk/blob/a29639715f47d7d5cab7d7aa667581bb1030620f/packages/core/src/CML/coreToCml/coreToCml.ts#L112

:thinking:

Reply:
> We've had the convention of managing any objects within the coreTo* translators. Can it not be done using the scope passed in?

Good point and yes the `txOut` and `tokenMap` scope.manage in this PR seems redundant. Then the only functional change in this PR seems to be `scope.manage(CML.min_ada_required(`. @mirceahasegan could you please test to verify?